lib/packages/docutils/rst.nim

```diff
@@ -626,10 +671,14 @@ proc isInlineMarkupEnd(p: RstParser, markup: string): bool =
     if markup != "``" and prevTok(p).symbol == "\\":
       result = false
 
-proc isInlineMarkupStart(p: RstParser, markup: string): bool =
+proc isInlineMarkupStart(p: RstParser, markup: string, twoTokens=false): bool =
   # rst rules: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#inline-markup-recognition-rules
   var d: char
-  result = currentTok(p).symbol == markup
+  if not twoTokens:
+    result = currentTok(p).symbol == markup
+  else:
+    result = currentTok(p).symbol == markup[0..0] and
```

Comment:
Why these strange-looking ranges here?

Reply:
it was a way to convert char->string. Changed to more conventional style.